src/Core/Types/FunctionType.cs

```diff
@@ -69,9 +69,34 @@ public FunctionType(ProcedureSignature sig) : base()
             }
         }
 
-        public FunctionType(SerializedSignature signature) : base()
+        static public FunctionType Create(ISerializedTypeVisitor<DataType> tldser, SerializedSignature signature)
```

Comment:
It seems like this method should be moved into the `TypeLibraryDeserializer.VisitSignature` class, since most of the work involves using the `tldser` variable.